apps/cli/src/commands/run.ts

```diff
@@ -71,28 +76,19 @@ export default class Run extends Command {
             projectName,
         ];
 
-        const mode = flags.verbose ? [] : ["--detach", "--wait"];
-
-        this.log("Starting node...");
         try {
             // run compose environment
-            await execa("docker", [...args, "up", ...mode], {
-                stdio: "inherit",
+            const up = execa("docker", [...args, "up"], {
```

Comment:
I tested this, and it kinda worked, but the message `?Press enter... ` doesn't show up. 

```suggestion
            const up = execa("docker", [...args, "up", "--attach", "validator"], {
```


```
+] Building 0.0s (0/0)                                                                                                                                                                                                                                                     
[+] Running 6/4
 ✔ Network 88dcfaab_default            Created                                                                                                                                                                                                                         0.1s 
 ✔ Container 88dcfaab-redis-1          Created                                                                                                                                                                                                                         0.0s 
 ✔ Container 88dcfaab-anvil-1          Created                                                                                                                                                                                                                         0.0s 
 ✔ Container 88dcfaab-database-1       Created                                                                                                                                                                                                                         0.0s 
 ✔ Container 88dcfaab-dapp_deployer-1  Created                                                                                                                                                                                                                         0.0s 
 ✔ Container 88dcfaab-validator-1      Created                                                                                                                                                                                                                         0.0s 
Attaching to 88dcfaab-validator-1
88dcfaab-validator-1  | GraphQL server listening at http://localhost:8080/graphql
88dcfaab-validator-1  | Inspect server listening at http://localhost:8080/inspect
88dcfaab-validator-1  | 2023-06-06 21:01:47.141826 info remote-machine pid:232 ppid:75 Initializing server on localhost:0
88dcfaab-validator-1  | time="2023-06-06T21:01:47Z" level=info msg="Configuration loaded from file: /etc/traefik/traefik.yaml"

```

Reply:
Yeah, this was not nice. I remove the message and the wait.
And then I'll investigate further the SIGTERM situation.

Reply:
Don't need the `"--attach", "validator"`.
I fixed an issue, it had a detached true, before.

Reply:
But with `--attach validator` you get logs only for the validator, and hides logs for the others, and have CTRL + C working.

Reply:
I'm wrong about CTRL + C.

Reply:
Hmm, that's true (about the log). Do you thing that's better?

Reply:
And do you think verbose mode should attach to all?

Reply:
Yes.

You can still call `docker logs -f` for other containers if needed.